src/main/java/com/google/sps/data/User.java

```diff
@@ -17,11 +30,26 @@ public User(MapLocation geolocation) {
   * @param zipcode
   * @return User with zipCode
   **/
-  public User(int zipCode) {
-
+  public User(String zipCode) {
+    this.geolocation = zipCodeToMapLocation(zipCode);
   }
 
   public MapLocation getGeolocation() {
     return geolocation;
   }
+
+  private MapLocation zipCodeToMapLocation(String zipCode) {
+    final GeoApiContext context = new GeoApiContext.Builder()
+            .apiKey(KEY)
+            .build();
```

Comment:
Is there a reason these lines are indented like this?

Reply:
The [style guide](https://google.github.io/styleguide/javaguide.html#s4.1-braces) says to use 4+ spaces when separating by the dot operator. I think it just makes it easier to read

Reply:
Ah okay, cool